DotNet/emitter.js

```diff
@@ -525,7 +549,9 @@
 				addHook('beforeEach', dot_net`initEyes("desktop", ScreenOrientation.Portrait);`)
 				break;
 			default:
-				throw Error(`Couldn't intrpret baselineName ${test.config.baselineName}. Code update is needed`)
+				//throw Error(`Couldn't intrpret baselineName ${test.config.baselineName}. Code update is needed`)
+				addHook('beforeEach', dot_net`initEyes("mobile", ScreenOrientation.Landscape);`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
				addHook("beforeEach", dot_net`initEyes("mobile", ScreenOrientation.Landscape);`)
```